jobs/frontend/senior-frontend-developer.md

```diff
@@ -0,0 +1,58 @@
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## We expect that you
+
+- Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important
+- Can design a whole system of application
+- Have experience working with different kind of teams and projects
+- Research new technologies and promote them inside the team (new libraries, tools, techniques: refactoring, debugging, etc.)
+- Will constantly suggest innovations on the project
+- It will be a plus if you released own projects
+- Understand yourself: have good habits, know how to be productive and how to rest
+- Have knowledge of English on the level which allows you to feel comfortable communicating in it (verbally and written) the whole working day.
+- Know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication. It is important for us.
+
+It will be a plus if you keen to become a Team Lead or already have this experience and know what skills are necessary to be a good Team Lead.
+
+## About us
+
+[![datarockets team](https://github.com/datarockets/career/blob/master/images/photo%20from%20New%20Year%20party.3.jpg)](https://www.instagram.com/datarockets/)
+
+We work remotely [#remote](https://github.com/datarockets/career#remote) and know how to do it well.
+
+We care about people [#people-first](https://github.com/datarockets/career#people-first) and projects [#work-as-a-team-with-clients](https://github.com/datarockets/career#work-as-a-team-with-clients).
+
+We build mobile and web apps for our clients. See [case studies](https://datarockets.com/case-studies/). Usually we work with clients for years and work as a single team.
+
+## Perks
+
+- 50% compensation for self-development activities: courses, conferences, books, educational platforms, English, sports.
+- Remote work and flexible schedule (actually no schedule at all - you decide when you work).
+- 32 paid days off per year - you can use them for vacations, occasional days off, and public holidays, only workdays count.
+- Full coverage of sick leaves.
+- 200 USD per year for medical expenses.
+- 500 USD per year for home office equipment.
+
+## How to apply 💌
+
+“Talk is cheap. Show me the code.” ― Linus Torvalds
+
+In order to better determine your current level and therefore increase your chances, we need to see your codes. If you already have some open-source projects that shows your competence then you can send them to us. Alternatevely, you can complete a [test task](./test-task.md). You might be even more interesting in that because completing the test task is a good way to add one more project to your portfolio and improve your skills.
```

Comment:
```suggestion
In order to better determine your current level and therefore increase your chances, we need to see your code. If you already have some open-source projects that shows your competence then you can send them to us. Alternatively, you can complete a [test task](./test-task.md). You might be even more interested in that because completing the test task is a good way to add one more project to your portfolio and improve your skills.
```